config/sync/metatag.metatag_defaults.node__event.yml

```diff
@@ -0,0 +1,16 @@
+uuid: 7f794961-e37a-432b-8ae5-8de3aa121032
+langcode: en
+status: true
+dependencies: {  }
+id: node__event
+label: 'Content: Event'
+tags:
+  schema_event_end_date: '[node:field_event_date:end_date:html_datetime]'
+  schema_event_url: '[node:event_url]'
+  schema_event_image: 'a:5:{s:5:"@type";s:11:"ImageObject";s:20:"representativeOfPage";s:4:"True";s:3:"url";s:34:"[node:field_event_logo:entity:url]";s:5:"width";s:29:"[node:field_event_logo:width]";s:6:"height";s:30:"[node:field_event_logo:height]";}'
+  schema_event_location: 'a:5:{s:5:"@type";s:5:"Place";s:4:"name";s:21:"[node:field_location]";s:3:"url";s:40:"[node:field_event_online_location:0:uri]";s:7:"address";a:2:{s:5:"@type";s:13:"PostalAddress";s:13:"streetAddress";s:21:"[node:field_location]";}s:3:"geo";a:3:{s:5:"@type";s:14:"GeoCoordinates";s:8:"latitude";s:34:"[node:field_event_coordinates:lat]";s:9:"longitude";s:34:"[node:field_event_coordinates:lon]";}}'
```

Comment:
This gives a validation error for events that are purely online and do not have a physical location:

> A value for the address field is required.

I see in the [location](https://schema.org/location) schema indeed that this is intended for physical locations, not for "online locations". There is no other field that is more suitable at this time.

I searched the feature requests for schema.org and it appears that online events is currently being worked on but it seems progress is slow recently. Ref. https://github.com/schemaorg/schemaorg/issues/1842

I think for the moment we cannot improve on this. Maybe our best shot is to file a followup to revisit this once online events are included in schema.org.